### PR TITLE
Update order-transactions.md

### PR DESCRIPTION
**Pull request by mschmele**
This seems like a reasonable endpoint for a "Process Purchase" endpoint

### CODE REVIEW

File: order-transactions.md [hunk 1]
```diff
@@ -23,6 +23,33 @@
 | merchant_defined_data_3 | ? | ? |
 | merchant_defined_data_4 | ? | ? |
 
+### Purchase items in current cart
+
+```http
+POST /order_transactions HTTP/1.1
+Accept: application/vnd.api+json
```
Comment (scottgonzalez) [inline order-transactions.md]: Missing `Content-Type` header.

File: order-transactions.md [hunk 2]
```diff
@@ -23,6 +23,33 @@
...
+```http
+POST /order_transactions
+Content-Type: application/vnd.api+json
+Accept: application/vnd.api+json
+
+{
+	"data": {
+		"type": "payment_method",
+		"attributes": {
+			"id": "123"
```
Comment (scottgonzalez) [inline order-transactions.md]: `id` is top level, not inside `attributes`. It's also not provided by the client for creation.

File: order-transactions.md [hunk 3]
```diff
@@ -23,6 +23,33 @@
...
+		},
```
Comment (scottgonzalez) [inline order-transactions.md]: Why is there no information about the actual transaction? How do we associate this with the actual transaction from CyberSource?

File: order-transactions.md [hunk 4]
```diff
@@ -23,6 +23,33 @@
...
+		"relationships": {
+			"user": {
```
Comment (scottgonzalez) [inline order-transactions.md]: This needs to be automatic. A user cannot create a transaction for a different user.

File: order-transactions.md [hunk 5]
```diff
@@ -23,6 +23,33 @@
 | merchant_defined_data_3 | ? | ? |
 | merchant_defined_data_4 | ? | ? |
 
+### Purchase items in current cart
+
+```http
+POST /order_transactions HTTP/1.1
+Accept: application/vnd.api+json
+Content-Type: application/vnd.api+json
```
Comment (mschmele) [inline order-transactions.md]: It was hiding right here the whole time

**Comment by mschmele** [comment]
Somehow I can still see this ¯\_(ツ)_/¯

**Comment by jflewis** [comment]
Hi mark

**Pull request closed without merging**